schedule_divoc.py

```diff
@@ -91,6 +92,10 @@ def main():
     print('  version: ' + full_schedule.version())
     print('  contains {events_count} events, with local ids from {min_id} to {max_id}'.format(**full_schedule.stats.__dict__))
 
+    repo = gitlib.Repo(search_parent_directories=True)
+    git_hash = repo.head.object.hexsha
+    full_schedule._schedule['schedule']['generator'] = git_hash
```

Comment:
```suggestion
    full_schedule._schedule['schedule']['generator'] = { "name": "voc/schedule/divoc", "version": git_hash }
```

Reply:
This should be enough for now... maybe check that the XML generator in voc/schedule.py converts it correctly...

Reply:
:+1: Thanks. Changed in 92c9c34ca13a257b6870efcdb928b547bff79d42.